diff --git a/out/manticoresearch-javascript/package.json b/out/manticoresearch-javascript/package.json
index 4a9f4ca..f8a13af
--- a/out/manticoresearch-javascript/package.json
+++ b/out/manticoresearch-javascript/package.json
@@ -1,6 +1,6 @@
 {
   "name": "manticoresearch",
-  "version": "3.2.0",
+  "version": "3.1.0",
   "description": "Javascript client for Manticore Search",
   "license": "MIT",
   "main": "src/index.js",
@@ -18,7 +18,8 @@
     "index"
   ],  
   "dependencies": {
-    "superagent": "5.1.0"
+    "superagent": "5.1.0",
+    "json-bigint": "1.0.0"
   },
   "devDependencies": {
     "inherits": "^2.0.4",
diff --git a/out/manticoresearch-javascript/src/ApiClient.js b/out/manticoresearch-javascript/src/ApiClient.js
index b6c1537..04b3a7e
--- a/out/manticoresearch-javascript/src/ApiClient.js
+++ b/out/manticoresearch-javascript/src/ApiClient.js
@@ -24,7 +24,7 @@
 
   /**
    * @module ApiClient
-   * @version 3.2.0
+   * @version 3.0.0
    */
 
   /**
@@ -335,6 +335,8 @@
     });
   };
 
+  exports.JSONbig = require('json-bigint')({ storeAsString: true });
+
   /**
    * Deserializes an HTTP response body into a value of the specified type.
    * @param {Object} response A SuperAgent response object.
@@ -348,14 +350,20 @@
     if (response == null || returnType == null || response.status == 204) {
       return null;
     }
-    // Rely on SuperAgent for parsing response body.
+    // Use json-bigint for parsing json responses otherwise rely on SuperAgent
     // See http://visionmedia.github.io/superagent/#parsing-response-bodies
-    var data = response.body;
+	var data;
+	if (returnType === Object || typeof returnType === 'object') {
+		data = exports.JSONbig.parse(response.text);
+	} else {
+		data = response.body;
+	}
     if (data == null || (typeof data === 'object' && typeof data.length === 'undefined' && !Object.keys(data).length)) {
       // SuperAgent does not always produce a body; use the unparsed response as a fallback
       data = response.text;
     }
-    return exports.convertToType(data, returnType);
+    const res = exports.convertToType(data, returnType);
+	return res;
   };
 
   /**
@@ -440,7 +448,8 @@
     }
 
     if (contentType === 'application/x-www-form-urlencoded') {
-      request.send(querystring.stringify(this.normalizeParams(formParams)));
+	  console.log('test request 2');
+	  request.send(querystring.stringify(this.normalizeParams(formParams)));
     } else if (contentType == 'multipart/form-data') {
       var _formParams = this.normalizeParams(formParams);
       for (var key in _formParams) {
