diff --git a/out/manticoresearch-java/src/main/java/com/manticoresearch/client/model/SqlResponse.java b/out/manticoresearch-java/src/main/java/com/manticoresearch/client/model/SqlResponse.java
index fd11fea..fd4731d 100644
--- a/out/manticoresearch-java/src/main/java/com/manticoresearch/client/model/SqlResponse.java
+++ b/out/manticoresearch-java/src/main/java/com/manticoresearch/client/model/SqlResponse.java
@@ -36,6 +36,7 @@ import java.util.logging.Logger;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.HashSet;
+import java.util.List;
 
 import com.fasterxml.jackson.core.JsonGenerator;
 import com.fasterxml.jackson.core.JsonParser;
