diff --git a/out/manticoresearch-java/src/main/java/com/manticoresearch/client/model/SourceRules.java b/out/manticoresearch-java/src/main/java/com/manticoresearch/client/model/SourceRules.java
index ab5d33c..ef93b38 100644
--- a/out/manticoresearch-java/src/main/java/com/manticoresearch/client/model/SourceRules.java
+++ b/out/manticoresearch-java/src/main/java/com/manticoresearch/client/model/SourceRules.java
@@ -25,6 +25,7 @@ import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonTypeName;
 import com.fasterxml.jackson.annotation.JsonValue;
+import java.util.ArrayList;
 import java.util.Arrays;
 import org.openapitools.jackson.nullable.JsonNullable;
 import com.fasterxml.jackson.annotation.JsonIgnore;
@@ -50,6 +51,9 @@ public class SourceRules {
   public static final String JSON_PROPERTY_EXCLUDES = "excludes";
   private JsonNullable<Object> excludes = JsonNullable.<Object>of([""]);
 
+  includes = JsonNullable.<Object>of(new ArrayList<>());
+  excludes = JsonNullable.<Object>of(new ArrayList<>(Arrays.asList("")));
+
   public SourceRules() { 
   }

   