diff --git a/out/manticoresearch-python/manticoresearch/rest.py b/out/manticoresearch-python/manticoresearch/rest.py
index a470462..325cb83 100644
--- a/out/manticoresearch-python/manticoresearch/rest.py
+++ b/out/manticoresearch-python/manticoresearch/rest.py
@@ -157,7 +157,7 @@ class RESTClientObject(object):
             if method in ['POST', 'PUT', 'PATCH', 'OPTIONS', 'DELETE']:
                 if query_params:
                     url += '?' + urlencode(query_params)
-                if re.search('json', headers['Content-Type'], re.IGNORECASE):
+                if headers['Content-Type'] == 'application/json':
                     request_body = None
                     if body is not None:
                         request_body = json.dumps(body)
