diff --git a/out/manticoresearch-python/manticoresearch/rest.py b/out/manticoresearch-python/manticoresearch/rest.py
index 2d9f935..9a2410b 100644
--- a/out/manticoresearch-python/manticoresearch/rest.py
+++ b/out/manticoresearch-python/manticoresearch/rest.py
@@ -177,6 +177,6 @@ class RESTClientObject:
                 if (
                     not content_type
-                    or re.search('json', content_type, re.IGNORECASE)
+                    or headers['Content-Type'] == 'application/json'
                 ):
                     request_body = None
                     if body is not None:
                     