diff --git a/out/manticoresearch-python/manticoresearch/rest.py b/out/manticoresearch-python/manticoresearch/rest.py
index a470462..325cb83 100644
--- a/out/manticoresearch-python/manticoresearch/rest.py
+++ b/out/manticoresearch-python/manticoresearch/rest.py
@@ -178,5 +178,5 @@ class RESTClientObject:
                content_type = headers.get('Content-Type')
                if (
                    not content_type
-                   or re.search('json', content_type, re.IGNORECASE)
+                   or headers['Content-Type'] == 'application/json'
