diff --git a/out/manticore-python/openapi_client/rest.py b/out/manticore-python/openapi_client/rest.py
index e3fe37d..8f56be0 100644
--- a/out/manticore-python/openapi_client/rest.py
+++ b/out/manticore-python/openapi_client/rest.py
@@ -157,7 +157,7 @@ class RESTClientObject(object):
             if method in ['POST', 'PUT', 'PATCH', 'OPTIONS', 'DELETE']:
                 if query_params:
                     url += '?' + urlencode(query_params)
-                if re.search('json', headers['Content-Type'], re.IGNORECASE):
+                if headers['Content-Type'] == 'application/json':
                     request_body = None
                     if body is not None:
                         request_body = json.dumps(body)
